### DISCUSSION TITLE
升级后专业版变免费版，帮解决一下，谢谢！

### DISCUSSION THREAD

**Discussion by cjhfly** [getlantern/lantern-forum-cn#381] [General]
![微信图片_20240403223538](https://github.com/getlantern/lantern-forum-cn/assets/25000529/8cd46122-d021-4194-afbd-4e4a58f7ff26)
![微信图片_20240403223538](https://github.com/getlantern/lantern-forum-cn/assets/25000529/5ecf99de-1170-41e6-8472-7c223e71f91c)

**Comment by cjhfly** [comment]
我邮箱  chenjianhui@126.com

**Comment by MaxMKMax** [comment]
@cjhfly 您好! 非常抱歉因为系统上出了一些问题, 我们正在修复了. 
我们暂时会通过手动帮您升级或发送激活码. 
非常抱歉也谢谢您支持蓝灯!

**Comment by MaxMKMax** [comment]
@cjhfly 您好. 支付部门说您尝试使用电子邮件链接重新登录就可以用了. 不需要激活码!

**Comment by cjhfly** [comment]
已授权，非常感谢！



在 2024-04-06 00:05:17，"Mike K" ***@***.***> 写道：

@cjhfly 您好. 支付部门说您尝试使用电子邮件链接重新登录就可以用了. 不需要激活码!

**Reply by MaxMKMax** [reply]
@cjhfly 您客气了. 谢谢您支持蓝灯!
